quickwit-core/src/indexing/index.rs

```diff
@@ -89,8 +92,13 @@ pub async fn index_data(
         .await?;
     }
 
-    if params.input_uri.is_none() {
-        println!("Please enter your new line delimited json documents.");
```

Comment:
after rebase this logic happens in the main.rs file, so you don't need to put any printing or stdout logic in here.
(as always separate the concern)